pkgs/tools/networking/grpc_cli/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, stdenv, fetchFromGitHub, automake, cmake, autoconf, curl, numactl }:
+
+stdenv.mkDerivation rec {
+  pname = "grpc_cli";
+  version = "1.46.6";
+
+  buildInputs = [ automake cmake autoconf curl numactl ];
+  src = fetchFromGitHub {
+    owner = "grpc";
+    repo = "grpc";
+    rev = "18dda3c586b2607d8daead6b97922e59d867bb7d";
+    sha256 = "sha256-m55FKwqlAQ2Ge7y4RMFO4W1onu0NYARWrOygChJMxng=";
+    fetchSubmodules = true;
+  };
+  configurePhase = "true";
+  buildPhase = ''
+    mkdir -p $out/bin
+    mkdir -p cmake/build
+    cd cmake/build
+    cmake -DgRPC_BUILD_TESTS=ON ../..
+    make grpc_cli
+    cp grpc_cli $out/bin
```

Comment:
This should probably belong to installPhase?

Reply:
Good idea, moved to install phase.